src/main/kotlin/no/nav/syfo/varsel/VarselServiceV2.kt

```diff
@@ -57,6 +54,32 @@ class VarselServiceV2 @Inject constructor(
         }
     }
 
+    fun ferdigstillSvarMotebehovVarselForNarmesteLederOgArbeidstaker(ansattFnr: String, virksomhetsnummer: String? = null) {
+
+        ferdigstillSvarMotebehovVarselForArbeidstaker(ansattFnr)
+
+        val narmesteLederRelations = narmesteLederService.getAllNarmesteLederRelations(ansattFnr)
+
+        narmesteLederRelations?.forEach {
+            if (virksomhetsnummer == null || virksomhetsnummer.equals(it.virksomhetsnummer)) {
```

Comment:
Stemmer det at virksomhetsnummer kan være null i input til denne funksjonen, men ikke i NL-relasjon? Er dette en bug eller feature? :P

Reply:
Ja, det stemmer at input kan være null, men ikke i NL-relasjonen. Det var forsøk på en feature, men det ble kanskje litt for subtilt.

Når en sykmeldt slutter å være kandidat til dialogmøte skal vi ferdigstille varsel til den sykmeldte og alle hens nærmeste ledere. I det tilfelle vil virksomhetsnummer være null.

Når en sykmeldt og en nærmeste leder blir kalt inn til dialogmøte, skal vi kun ferdigstille varselet for lederen i bedriften som er innkalt.

Jeg skal se om jeg greier å gjøre det litt mer forståelig.

Reply:
Kan du sjekke ut e2a8fc6d570568cfa03213368243fd3cf460b92e og se om det ble bedre?